### PR TITLE
Use string value of Enums when looking them up

### PR DESCRIPTION
**Pull request by bdemers**

### CODE REVIEW

**Comment by bdemers** [comment]
Going to release 0.8.1 after this is merged
Fixes #125

**Comment by bdemers** [comment]
**NOTE**: looks like previous coverage run triggered a http request retry (not related to this PR)

File: impl/src/main/java/com/okta/sdk/impl/resource/EnumConverter.java [hunk 1]
```diff
@@ -0,0 +1,71 @@
...
+     * @return the enum constant of the specified enum type with the specified value
+     */
+    <E extends Enum<E>> E fromValue(Class<E> enumType, String value) {
+
+        Assert.notNull(value, "[Assertion failed] - 'value' is required; it must not be null");
+
+        // Get the reverse enum map
+        Map<String, E> reverseMap = getReverseLookupMap(enumType);
+
+        //lookup the value
```
Comment (jmelberg-okta) [inline impl/src/main/java/com/okta/sdk/impl/resource/EnumConverter.java]: nit: Spacing after `//`

File: pom.xml [hunk 2]
```diff
@@ -20,7 +20,7 @@
     <parent>
         <groupId>com.okta</groupId>
         <artifactId>okta-parent</artifactId>
-        <version>1</version>
+        <version>2</version>
```
Comment (jmelberg-okta) [inline pom.xml]: Curious: Do you need to specify a `SNAPSHOT` tag?

<https://github.com/okta/okta-java-parent/blob/master/pom.xml#L28>

Or is this convention?

Comment (bdemers) [reply pom.xml]: v2 is already released

**Pull request merged**